### ISSUE TITLE
Which loss to consider when monitoring training with cosine-softmax loss mode?

### ISSUE THREAD

**Issue by samuelsciberras** [nwojke/cosine_metric_learning#91]
I am training a custom dataset using cosine-softmax, but all triplet loss, magnet loss, and cross entropy loss are being shown on tensorboard. 

Which one should I be looking at when using cosine-softmax loss mode? I understand that cosine-softmax minimizes cross entropy loss, so it should be that graph, but what I cannot understand is why the other graphs are also being adjusted.

Also, what is the recommended amount of data for training?

Thanks

**Comment by sherlockchou86** [comment]
https://github.com/nwojke/cosine_metric_learning/blob/master/train_app.py#L651

maybe you can set monitor_xx as False when using cosine-softmax loss mode. the code can also monitor other loss value when using cosine-softmax mode.
